tests/hypothesis_strategies/kernels.py

```diff
@@ -2,13 +2,29 @@
 
 import hypothesis.strategies as st
 
-from baybe.kernels import MaternKernel
+from baybe.kernels import MaternKernel, ScaleKernel
 
 from ..hypothesis_strategies.priors import priors
 
 matern_kernels = st.builds(
     MaternKernel,
     nu=st.sampled_from((0.5, 1.5, 2.5)),
-    lengthscale_prior=st.one_of(st.none(), priors),
+    lengthscale_prior=priors,
+    lengthscale_prior_initial_value=st.floats(min_value=0, exclude_min=True),
 )
-"""A strategy that generates Matern kernels."""
+"""A strategy that generates matern kernels."""
+
+
+base_kernels = st.one_of([matern_kernels])
+"""A strategy that generates base kernels to be used within more complex kernels."""
+
+scale_kernels = st.builds(
```

Comment:
From @Scienfitz: What about the ScaleKernel built as composite ?

Reply:
Did that, is this now as it should be? If so please resolve.

Reply:
we still have the weird thing that definitions like that are not strategies and need to be called with `()`, to fix it the actual definition should go into `_scale_kernels` with an added `scale_kernels=_scale_kernels()` below.

But on another note: I was more thinking of making the `kernels` a composite already here. Since it would mean `scale_kernels` would be more or less obsolete

Reply:
fixed